### PR TITLE
Add public logs

### PR DESCRIPTION
**Pull request by vytis**
## What's new in this PR?

To debug problems in production builds we want to have some amount of logging. This PR adds support for writing anonymized information to logs.

### CODE REVIEW

**Comment by typfel** [comment]
Let's wait until the release build is finished

**Comment by typfel** [comment]
Release is done, let's merge this?

File: Wire-iOS/Sources/Components/Settings+Logging.swift [hunk 1]
```diff
@@ -45,18 +45,14 @@ extension Settings {
     
     /// Loads from user default the list of logs that are enabled
     @objc public func loadEnabledLogs() {
-        let avsTag = "AVS"
+        var tagsToEnable = Set(arrayLiteral: "AVS", "Network", "SessionManager", "Conversations", "calling", "link previews", "event-processing", "SyncStatus", "OperationStatus", "Push", "Crypto", "cryptobox")
```
Comment (alexaubry) [inline Wire-iOS/Sources/Components/Settings+Logging.swift]: Why not use the array literal directly?

```suggestion
        var tagsToEnable: Set<String> = ["AVS", "Network", "SessionManager", "Conversations", "calling", "link previews", "event-processing", "SyncStatus", "OperationStatus", "Push", "Crypto", "cryptobox"]
```

Comment (vytis) [reply Wire-iOS/Sources/Components/Settings+Logging.swift]: 🤷‍♂

**Pull request merged**